src/java/org/apache/cassandra/net/MessagingService.java

```diff
@@ -224,23 +224,6 @@ public enum Version
         VERSION_50(13),
         VERSION_51(14);
 
-        public static final Version CURRENT;
```

Comment:
documented why i removed this in `current()`, but the jist is that this is now touched by `Config` so touching DD adds a circular dependecy